### PR TITLE
 Use Gradle 5.3.1

### PR DESCRIPTION
**Pull request by rpalcolea**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jdk:
 env:
   matrix:
     - GRADLE_VERSION=default
-    - GRADLE_VERSION=5.2.1
+    - GRADLE_VERSION=5.3.1
```
Comment (KengoTODA) [inline .travis.yml]: To keep test with 5.2, please do not replace existing lines, instead add a new line for `5.3.1`

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.3.1-bin.zip
```
Comment (KengoTODA) [inline gradle/wrapper/gradle-wrapper.properties]: You do not need to update this file, see `.travis.yml` for detail.

Comment (rpalcolea) [reply gradle/wrapper/gradle-wrapper.properties]: got it, done! 👍

File: .travis.yml (hunk 1 above)
Comment (rpalcolea) [reply .travis.yml]: got it, done! 👍

**Comment by KengoTODA** [comment]
@rpalcolea could you merge `origin/master` to your topic branch? It will fix the build failure in this PR.

**Comment by rpalcolea** [comment]
done @KengoTODA

**Comment by KengoTODA** [comment]
wmm, I'm not sure why the CI fails... it seems that Travis CI's feature is not working in intuitive way. I'm sure that your change isn't root cause, but please wait for a while.

**Comment by KengoTODA** [comment]
@rpalcolea Thanks for your waiting. Could you merge origin/master to your topic branch again?

**Comment by rpalcolea** [comment]
Hi @KengoTODA , done!

btw, are there any plans for a new release soon? would be great to get this PR released since it would remove deprecation warnings in builds -> https://github.com/spotbugs/spotbugs-gradle-plugin/pull/134

**Comment by KengoTODA** [comment]
Not sure when we release next version. I want to fix some of https://github.com/spotbugs/spotbugs-gradle-plugin/milestone/3 but we have no contribution recently.

**Pull request merged**